index.html

```diff
@@ -1738,88 +1738,65 @@
 Link this to [[[#guardians]]].
 </aside>
 
-## Harassment
+## Abuse
+
+<div class="practice">
+  <p>
+    <span class="practicelab" id="abuse-reporting">
+      Systems that allow for communicating on the Web must provide an
+      effective capability to report abuse. [=User agents=] and [=sites=] must
+      take steps to protect their users from abusive behaviour, and abuse
+      mitigation must be considered when designing web platform features.
+    </span>
+  </p>
+</div>
 
 Online <dfn>harassment</dfn> is the "pervasive or severe targeting of an individual or group online
-through harmful behavior" [[PEN-Harassment]]. Harassment is a prevalent problem on the Web,
-particularly via social media. While harassment may affect any person using the Web, it may be more
+through harmful behavior" [[PEN-Harassment]]. Harassment is a prevalent problem on the web,
+particularly via social media. While harassment may affect any person using the web, it may be more
 severe and its consequences more impactful for LGBTQ people, women, people in racial or ethnic
 minorities, people with disabilities, [=vulnerable people=] and other marginalized groups.
 
-<aside class="note">
-  Some useful research overviews of online harassment include: [[?PEW-Harassment]],
-  [[?Addressing-Cyber-Harassment]] and [[?Internet-of-Garbage]].
-</aside>
-
-[=Harassment=] is both a violation of privacy itself and can be magnified or facilitated by other
-violations of privacy.
+[=Harassment=] is both a violation of privacy itself and can be enabled or
+exacerbated by other violations of privacy.
 
-Abusive online behavior may include: sending [=unwanted information=]; directing others to contact
-or bother a person ("dogpiling"); disclosing sensitive information about a person; posting false
-information about a person; impersonating a person; insults; threats; and hateful or demeaning
-speech.
+Harassment may include: sending [=unwanted information=]; directing others to contact
+or bother a person ("dogpiling"); disclosing [sensitive information](#sensitive-information) about a person; posting false information about a person; impersonating a person; insults; threats; and hateful or demeaning speech.
 
-Disclosure of identifying or contact information (including "doxxing") can be used, including by
-additional attackers, to send often persistent unwanted information that amounts to harassment.
-Disclosure of location information can be used, including by additional attackers, to intrude on a
+Disclosure of identifying or contact information (including "doxxing") can be used to send often persistent [=unwanted information=] that amounts to harassment.
```

Comment:
"send often persistent" read funny, so maybe:

```suggestion
Disclosure of identifying or contact information (including "doxxing") can often be used to cause additional attackers to send persistent [=unwanted information=] that amounts to harassment.
```